### ISSUE TITLE
Remove `dagre` as it's no longer maintained

### ISSUE THREAD

**Issue by edewit** [keycloak/keycloak#17774]
We have a [dependency](https://github.com/keycloak/keycloak-admin-ui/blob/8196a44612c3922e95ea324a4dc0b4d161256630/package.json#L32) on [`dragre`](https://github.com/dagrejs/dagre), which is no longer maintained. This issue is to remove the dependency and replace it with something better.

**Comment by akbar1214** [comment]
@edewit Seems like [Dagre](https://github.com/dagrejs/dagre) dependency is maintained. The last PR merge was around 2 months ago and there are releases after creation of the ticket.

I think updating the package.json to use [@dagrejs/dagre](https://www.npmjs.com/package/@dagrejs/dagre) instead of [dagre](https://www.npmjs.com/package/dagre) should work (Not a FE expert). 

WDYT ?

**Comment by jonkoops** [comment]
Yes, that would be an option. Someone would have to do a sanity check to see if there are any breaking changes. I created #37467 to resolve this in case anyone is interested in picking this up, so I will close this issue. Thanks @akbar1214.
